modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-web/src/main/java/com/liferay/portal/workflow/kaleo/forms/web/internal/display/context/KaleoFormsAdminDisplayContext.java

```diff
@@ -401,12 +403,10 @@ _renderRequest, _getIteratorURL(), null,
 					emptyResultsMessage);
 
 			searchContainer.setResultsAndTotal(
-				() ->
-					WorkflowDefinitionManagerUtil.getActiveWorkflowDefinitions(
-						_themeDisplay.getCompanyId(),
-						searchContainer.getStart(), searchContainer.getEnd(),
-						null),
-				WorkflowDefinitionManagerUtil.getActiveWorkflowDefinitionsCount(
+				() -> _workflowDefinitionManager.getActiveWorkflowDefinitions(
```

Comment:
@JanisZhang The change to this class doesn't seem to be related to `KaleoFormsUtil`, please put it in a separate commit.